.docker/readme.md

```diff
@@ -143,15 +143,27 @@ Note: All of these commands are meant to be executed from project directory.
 Once the Docker instance has started, you can visit [2014.content.wordcamp.org](2014.content.wordcamp.org) to view a sample WordCamp site. WordCamp central would be [central.wordcamp.test](central.wordcamp.test). You can also visit [localhost:1080](localhost:1080) to view the MailCatcher dashboard.
 
 
-## Updating the sample database
+## Working with the database provision file
 
-1. Make sure WP is running the latest branch, and the database schema has been updated.
-1. Make sure there isn't anything sensitive in the database. Scrub anything that is.
-1. Update the sample file:
+The **.docker/bin** directory gets mounted as a volume within the PHP container, and it contains a script, **database.sh**, with several useful commands. To run these commands, first open a shell inside the docker container:
 
-	```bash
-	docker-compose exec wordcamp.db bash
-	mysqldump wordcamp_dev -u root -pmysql > /var/lib/mysql/wordcamp_dev.sql
-	exit
-	mv .docker/database/wordcamp_dev.sql .docker/wordcamp_dev.sql
-	```
+    ```bash
+    docker-compose exec wordcamp.test bash
+    ```
+
+From there you can run the script using `bash /var/scripts/database.sh [subcommand]`. The most useful subcommands are:
+
+* `backup-current`: Back up the current state of the database to a file in the **.docker/data** directory.
+* `restore <file>`: Import a specified file to the database after deleting the current state.
+* `reset`: Delete the current state of the database and then re-import the provision file.
+* `clean-export`: Clean up the current state of the database and then export it to the provision file. See more on this below.
+* `help`: See the full list of subcommands available.
+
+### Updating the database provision file
+
+If the dev database needs to be updated to better reflect the state of production (e.g. a new version of the WP Core database, new network-activated plugins), or you've added data to your dev database that you think should be included for everyone (e.g. a new site with useful test cases), you can use the `clean-export` subcommand to update the file that is committed to version control. Before you do, please do these preflight checks:
+
+* Make sure WP is running the latest branch, and the database schema has been updated.
+* Make sure there isn't anything sensitive in the database. Scrub anything that is.
```

Comment:
```suggestion
* Review each line of the diff to make sure there isn't anything sensitive in the database. Scrub anything that is.
```

`MIGHT` - It might be good to make this a bit more strict/explicit, just to make it clear how important that is. It might also help to provide some guidance on making that less inconvenient. [The Meta Environment has some tips there](https://github.com/WordPress/meta-environment/wiki/Reviewing-PRs-with-database-changes), but they're still pretty difficult to do. I'm not sure if there's a better way, though.